scripts/Packages/Package_Gullwing__QFP_SOIC_SO/Readme.md

```diff
@@ -0,0 +1,151 @@
+# IPC gullwing generator
+
+This generator uses IPC-7351B equations and fillet definitions to derive a footprint for gullwing style packages. (fillet definitions can be changed by pointing to a different definition file.) The script generates footprints in zero orientation version A (pin 1 at top left corner).
```

Comment:
Missing a capital 'F' in the sentence inside the parens.

Reply:
Is it really necessary to put the sentence regarding to fillet definitions in brackets?

Reply:
both fixed

Reply:
Now the sentence inside the parens is captalized, but it's not a stand-alone sentence. Add a period to close the preceding sentence and move the period for the paren sentence inside the paren or else don't capitalize the phrase inside the parents. For clarity, the first solution looks like: `This generator uses IPC-7351B equations and fillet definitions to derive a footprint for gullwing style packages from the dimensions of the package. (The suggested footprint is only used to determine the size of the exposed pad.)`

Reply:
done